### PR TITLE
footer

### PR DESCRIPTION
**Pull request by smplacc**
New Footer

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,4 +9,5 @@
 <body>
     Hello world
 </body>
+Footer
```
Comment (Algol22) [inline index.html:12]: Fine

File: index.html [hunk 2]
```diff
@@ -9,4 +9,5 @@
 <body>
     Hello world
```
Comment (Algol22) [inline index.html:10]: update later

Comment (smplacc) [reply index.html:10]: Thank you

File: index.html (hunk 1 above)
Comment (smplacc) [reply index.html:12]: Thx

**Pull request merged**